tour/beerDEcoded.yaml

```diff
@@ -0,0 +1,188 @@
+      these bacteria are actually very good for us. And some others can make us
+      very ill. Bacteria come in different shapes and sizes but they have the
+      same components. One crucial component is the DNA, the blueprint of life.
+      The DNA encodes the shape and size and many other things unique for a
+      bacterial species. Because of the encoding information the DNA can be used
+      to identify what kind of bacteria the DNA is from. Therefore, within a
+      sample form soil, our gut or beer one can specify what kind of species are
+      inside the sample. Follow this tour to learn more about this kind of
+      analysis. <br><br> Read and Follow the instructions before clicking
+      <b>'Next'</b>.<br><br> Click <b>'Prev'</b> in case you missed out on any
+      step."
+    backdrop: true
+  - title: <b>Create a new history</b>
+    element: '#history-new-button'
+    intro: >-
+      Let's start by creating a new history.<br>Click on the button <b>'Create
+      new history'</b><br>
+    position: left
+    preclick:
+      - '#center-panel'
+  - title: <b>Rename the history</b>
+    element: '#current-history-panel > div.controls'
+    intro: Change the name of the new history to <b>'BeerDEcoded'</b>.
+    position: left
+  - title: <b>Uploading the input data</b>
+    content: We need to upload data. We will start with a <b>fastq</b> file.
+    backdrop: true
+  - title: <b>Uploading the input data</b>
+    element: .upload-button
+    intro: >-
+      We will import fastq into the history we just created.<br><br> Click
+      <b>'Next'</b> and the tour will open Galaxy Upload Manager and take you to
+      the Upload screen.
+    position: right
+    postclick:
+      - .upload-button
+  - title: <b>Uploading the input data</b>
+    intro: Upload the data by clicking the <b>'Paste/Fetch Data'</b> button.
+  - title: <b>Uploading the input data</b>
+    content: >-
+      Load the data into your history by providing the links or choose your
+      fastq file from your computer. <br>Click on <b>'Start'</b> to upload the
+      data into your Galaxy history.
+  - title: <b>Uploading the input data</b>
+    content: >-
+      The upload may take awhile.<br><br> Hit the <b>Close</b> button when you
+      see that the files are uploaded into your history.
+  - title: <b>Uploading the input data Complete !</b>
+    content: 'Now that your data is ready, let''s use some tools.<br>'
+    backdrop: true
+  - title: <b>Assign taxonomic classifications</b>
+    content: >-
+      One of the key steps in metagenomic data analysis is to identify the taxon
+      to which the individual reads belong. Taxonomic classification tools are
+      based on microbial genome databases to identify the origin of each
+      sequence.<br>
+    backdrop: true
+  - title: <b>Taxonomic classification with Kraken2</b>
+    content: >-
+      To perform the taxonomic classification we will use Kraken2. This tool
```

Comment:
I suggest the explanation:

"This tool uses <a href="https://en.wikipedia.org/wiki/K-mer">k-mers</a> (the read’s subsequences of length k) to assign a taxonomic label to the sequence (if possible).
The taxonomic label is assigned based on matches of k-mer content of the considering sequence to the k-mer content of reference genome sequence. The result is a classification of the considering sequence to the most likely taxonomic label. If the k-mer content is not similar to any genomic sequence in the database used, it will not assign any taxonomic label.
"

Reply:
A figure would be nice. Maybe a pile of readers split in k-mers and then searched in the DB.

Reply:
![kraken2_simlified](https://user-images.githubusercontent.com/55543056/128420143-f5bc2a63-cc62-4491-a64d-a92d79262eae.jpg) - here I tried to create an approximate figure for simplified description of Kraken2 process

Reply:
This looks very nice. Thanks!